### DISCUSSION TITLE
Smart Connect v1.1.43

### DISCUSSION THREAD

**Discussion by fabioscarsi** [brianpetro/obsidian-smart-connections#562] [General]
What exactly is the innovation under Smart Connect v1.1.43?
Can we use Smart Connect to give our customGPTs access to our vaults?
How exactly? I've tried but something doesn't work for me. Any tutorial?
Anyway seems extremely interesting ....

**Comment by brianpetro** [comment]
Currently, only the official Smart Connect - Obsidian GPT is available. However, the ability to make your own custom GPT using the Smart Connect software to interact with your vault is just me-writing-instructions-on-how-to-do-it away. 

In other words, custom GPTs via Smart Connect are coming very soon, probably this week. 
🌴

**Reply by fabioscarsi** [reply]
Gorgeous!

**Reply by brianpetro** [reply]
@fabioscarsi there's a link to my first "getting started with custom GPTs using Smart Connect" video in the supporter chat at this link https://chat.smartconnections.app/rooms/5/@1066 🌴

**Reply by fabioscarsi** [reply]
I just set it up in my personal assistant CustomGPT. It works, great. Now, my personal assistant CustomGPT (that, for example, can access Todoist through their API) can also access my Obsidian Vault and work directly on my notes. I was waiting for this. Now I must teach it how to best interact with my vault in a RAG logic. I'll experiment. Great addition, thanks Brian!
